packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -242,25 +268,41 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     : horizontalGalleryParticipants.map((participant): JSX.Element => {
         return defaultOnRenderVideoTile(
           participant,
+          false,
           isNarrow ? SMALL_HORIZONTAL_GALLERY_TILE_STYLE : LARGE_HORIZONTAL_GALLERY_TILE_STYLE
         );
       });
 
+  const localScreenShareStreamComponent = <LocalScreenShare localParticipant={localParticipant} />;
+
+  const remoteScreenShareComponent = (
+    <RemoteScreenShare
+      screenShareParticipant={screenShareParticipant}
+      onCreateRemoteStreamView={onCreateRemoteStreamView}
+    />
+  );
+
   return (
-    <div ref={containerRef} className={videoGalleryOuterDivStyle}>
-      <Stack id={FLOATING_TILE_HOST_ID} grow styles={videoGalleryContainerStyle}>
-        {shouldFloatLocalVideo && (
-          <Modal
-            isOpen={true}
-            isModeless={true}
-            dragOptions={DRAG_OPTIONS}
-            styles={floatingLocalVideoModalStyle(theme, isNarrow)}
-            layerProps={{ hostId: FLOATING_TILE_HOST_ID }}
-          >
-            {localParticipant && localVideoTile}
-          </Modal>
+    <div id={FLOATING_TILE_HOST_ID} ref={containerRef} className={videoGalleryOuterDivStyle}>
```

Comment:
Something to consider but not necessary: To avoid raw `div` we might be able to use fluent's `Ref` instead here

Reply:
Do you mean [Ref](https://fluentsite.z22.web.core.windows.net/0.52.2/components/ref/definition) from fluentui-northstar? I could not find one for FluentUI. Also no responses yet on the question you asked [here](https://teams.microsoft.com/l/message/19:86b094239256467da9dfa96ba0897ca2@thread.skype/1635182270286?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ffe264f2-14d0-48b5-9384-64f808b81294&parentMessageId=1635182270286&teamName=Fluent%20Community&channelName=Fluent%20UI%20React%20(Web)&createdTime=1635182270286) from FluentUI.

Reply:
Ahh just noticed we used it in message thread, didn't notice it was northstar - dang
never mind then! :)